packages/flutter_tools/lib/src/devfs.dart

```diff
@@ -263,6 +264,13 @@ class DevFSException implements Exception {
   final StackTrace stackTrace;
 }
 
+HttpClient get httpClient {
```

Comment:
Can this be scoped down from a top-level public? If this is only used to initialize `_DevFSHttpWriter._client`, maybe it can be scoped down to the initializer for that field or to the constructor of `_DevFSHttpWriter`.

Reply:
Done